extensions/wikia/AdEngine/js/template/floating-rail.js

```diff
@@ -0,0 +1,93 @@
+/*global define*/
+define('ext.wikia.adEngine.template.floating-rail', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.uapContext',
+	'ext.wikia.adEngine.adHelper',
+	'jquery',
+	'wikia.log',
+	'wikia.document',
+	'wikia.window'
+], function (adContext, uapContext, adHelper, $, log, doc, win) {
+	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)